### ISSUE TITLE
Result quality is not comparable to the paper

### ISSUE THREAD

**Issue by ryushinn** [tflsguoyu/svbrdf-diff-renderer#2]
Hi,

Thanks for your great work! I am trying to run this repo but results seem less plausible than those in paper. 

Basically, I ran the `run_optim_ganlatent.py` script with the JSON config `data/card_blue/optim.json` and expected to get results reported in Fig.1 and Fig.13 in the paper, like below. 

![image](https://github.com/tflsguoyu/svbrdf-diff-renderer/assets/44772706/be8aeb2e-9c8f-4bc9-830a-e3fc7e706cd2)

But what I got was (optimizing for 300k epochs, i.e., 1-hour training in Nvidia 4090):

![tex](https://github.com/tflsguoyu/svbrdf-diff-renderer/assets/44772706/e71011f2-1b95-476b-873f-54c9c9469a44)

and the loss curve indicated the convergence:

![loss_curve copy](https://github.com/tflsguoyu/svbrdf-diff-renderer/assets/44772706/5970dc72-3449-4f7d-922a-88cbcf4af8f7)

I think the quality gap is because this script is still WIP and some important components haven't been implemented. For example, these might be missing (correct me if I were wrong, sorry):
1. Initialization as per section 4.4;
2. Loss function as per equation 3;
3. alternative optimization strategy as per section 4.3.

**Do you have any expected date to implement the full algorithm which can produce results at the same level of quality in the paper? Or would you mind giving any comments on how these missing components will affect the final SVBRDF reconstruction quality.**

I want to reproduce some of your results recently so your help will be highly appreciated. BTW, I also tried the [old repo](https://github.com/tflsguoyu/materialgan) but the download links of the data and the pretrained model expired. It would be also helpful if the old repo can produce comparable results to the paper, and you could bring it back somehow. :)

**Comment by tflsguoyu** [comment]
Sorry I am still working on the codes. Now `run_optim_ganlatent.py` only optimizes w+, and noise is still missing.
I will try my best to finish it ASAP. 
Thank you for your support!

**Comment by tflsguoyu** [comment]
The codes are updated, and I regenerated all the results for real captured data.
